src/actions/installation-repositories-added.ts

```diff
@@ -0,0 +1,133 @@
+import path from 'path'
+import mls from 'multilines'
+import * as probot from 'probot'
+import { promises as fs } from 'fs'
+
+import { base64, sha } from '../utils'
+import { getCollectiveWithGithubHandle } from '../collective'
+
+const BRANCH_NAME = `opencollective-bot/configuration`
+
+const FILE_PATH = '.github/opencollective.yml'
+
+const PR_TITLE = 'Open Collective Bot configuration'
+
+const PR_BODY = mls`
+  | Hi!
+  |
+  | Thank your for installing the Open Collective Bot.
+  |
+  | To activate and configure it, please review and merge this PR.`
+
+export default async (context: probot.Context) => {
+  const { github, payload } = context
+
+  for (const repositoryAdded of payload.repositories_added) {
+    const [owner, repo] = repositoryAdded.full_name.split('/')
+
+    // TEMP: don't commit
```

Comment:
You've been trolled by your past self

Reply:
Is commiting as a comment really commiting? :-)